web/cobrands/oxfordshire/js.js

```diff
@@ -75,4 +75,3 @@
         }
     }
 })();
-//*/
```

Comment:
I think this was put here on purpose, there's a similar one at the start of the file. I think they should either both be removed or both left in.